examples/cyclotron/input_cyclotron.in

```diff
@@ -3,45 +3,43 @@
 ###############################################################################
 beam.npart = 10000
 beam.units = static
-beam.energy = 2.0e3
+beam.energy = 4.0e-3
 beam.charge = 1.0e-9
-beam.particle = electron
+beam.particle = proton
 beam.distribution = waterbag
-beam.sigmaX = 3.9984884770e-5
-beam.sigmaY = 3.9984884770e-5
-beam.sigmaT = 1.0e-3
-beam.sigmaPx = 2.6623538760e-5
-beam.sigmaPy = 2.6623538760e-5
-beam.sigmaPt = 2.0e-3
-beam.muxpx = -0.846574929020762
-beam.muypy = 0.846574929020762
+beam.sigmaX = 1.0e-3
+beam.sigmaY = 1.0e-3
+beam.sigmaT = 0.3
+beam.sigmaPx = 2.0e-4
+beam.sigmaPy = 2.0e-4
+beam.sigmaPt = 2.0e-5
+beam.muxpx = 0.0
+beam.muypy = 0.0
 beam.mutpt = 0.0
 
 
 ###############################################################################
 # Beamline: lattice elements and segments
 ###############################################################################
-lattice.elements = half half
-lattice.periods = 1000
+lattice.elements = monitor half half monitor
```

Comment:
Do we want to make this
```suggestion
lattice.elements = monitor half monitor half
```
?

Otherwise it is for turns:
```
monitor half half monitor monitor half half monitor monitor half half monitor ...
```

With the change, it will not have a monitor at the last element though:
```
monitor half monitor half monitor half ... monitor half
```